video/out/vulkan/context_display.c

```diff
@@ -318,6 +398,20 @@ static bool display_init(struct ra_ctx *ctx)
         goto error;
     }
 
+#if HAVE_DRM
+        VkPhysicalDevicePCIBusInfoPropertiesEXT pci_props = {
```

Comment:
It's worth pointing out that this technically requires `VK_EXT_pci_bus_info` to be loaded. Which, incidentally, I just realized is completely bugged: the vulkan extension is listed as a device extension (and libplacebo loads it as such), but it's used on the physical device (which is an instance-level object).

Meaning it should actually be an instance extension. I'll raise it with khronos. For now, I guess, just ignore it, since it works, even though it's technically undefined behavior.

Reply:
I looked it up and the spec says that this is fine, you're allowed to query physical device properties of physical devices that have the appropriate device-level extension, without actually needing to *enable* that extension.

However, that does mean that for this to not be undefined behavior, you need to test if the `VkPhysicalDevice` you're looking at supports `VK_EXT_pci_bus_info`.

Reply:
I should probably add a field to `pl_vulkan_device_params` to allow you to require the presence of certain extensions on chosen vulkan devices. I'll fix this in libplacebo.

Reply:
Much thanks!

Reply:
@haasn What will this mean for the mpv side of things?

Reply:
It'll mean adding something like

```
#if PL_API_VER >= XYZ
    .extensions = &VK_EXT_PCI_BUS_INFO_EXTENSION_NAME,
    .num_extensions = 1,
```

On `vulkan_params`